docs/helpers/html.ts

```diff
@@ -12,7 +12,7 @@ export const html = (
     `<${name}`,
     Object.entries(props).map(toAttribute).join('\n'),
     '>',
-    children,
+    Array.isArray(children) ? children.join('') : children,
```

Comment:
I'm not sure about this, what do you think @rabelloo maybe better just each time to join on each use?